### PR TITLE
fix: broken markdown editor on `ark.io`

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

The issue is related to purgecss and/or webpack.mix removing some classes when imported through a JS file.

This PR moves all the classes related to the markdown to an external file that is correctly handled with laravel `webpack.min` so they can be imported on the pages where we have the markdown editor.

To test merge this on the two projects that use the markdown, recompile (in production) and test that everything works as expected:

1. Ark site: https://github.com/ArkEcosystem/ark.io/pull/972
2. Msq: https://github.com/ArkEcosystem/marketsquare.io/pull/2149

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [x] I added an explanation on how to use the component to the readme _(if necessary)_
-   [x] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/assets/css/markdown-editor.css [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
+/** UI Repo Styles **/
+@import "../../assets/css/styles-with-prism.css";
+
 /* purgecss start ignore */
+@import "@toast-ui/editor/dist/toastui-editor.css";
+@import "codemirror/lib/codemirror.css";
+
 .ark-markdown-editor .te-md-container .CodeMirror,
 .ark-markdown-editor .te-md-container .te-preview .tui-editor-contents,
 .ark-markdown-editor .tui-editor .te-md-splitter {
```
Comment (ItsANameToo) [inline resources/assets/css/markdown-editor.css:12]: why was this file renamed from `_markdown-editor` to `markdown-editor`?

File: resources/assets/css/markdown-editor.css [hunk 2]
```diff
@@ -1,4 +1,12 @@
+@import "_tailwind.css";
+
+/** UI Repo Styles **/
+@import "../../assets/css/styles-with-prism.css";
```
Comment (ItsANameToo) [inline resources/assets/css/markdown-editor.css:4]: why does it need to import tailwind and all the other styles? those will already be included in the project itself

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/assets/css/markdown-editor.css (hunk 1 above)
Comment (alfonsobries) [reply resources/assets/css/markdown-editor.css:12]: because the `markdown-editor` file is like another main file since it's used alone, I mean it's not imported inside another file, so the underscore that usually means it is a partial is not really accurate

File: resources/assets/css/markdown-editor.css (hunk 2 above)
Comment (alfonsobries) [reply resources/assets/css/markdown-editor.css]: @ItsANameToo hmm.. short answer: I wasn't thinking clearly, long: I was trying to make the compilation works and while trying some alternatives come up with that as a solution to handle the tailwind variables.

I really just needed to add the files that contained the variables used on  to the markdown, [fixed on this last commit](https://github.com/ArkEcosystem/laravel-ui/pull/556/commits/6e74c44f22d5530a473e5de9af107ec0c2796379)

**Comment by alfonsobries** [comment]
@ItsANameToo ✅  first comment answered, second one fixed

File: resources/assets/css/markdown-editor.css (hunk 1 above)
Comment (ItsANameToo) [reply resources/assets/css/markdown-editor.css:11]: I see. Need to check if we import this directly anywhere as that would mean a breaking change 🤔

Comment (alfonsobries) [reply resources/assets/css/markdown-editor.css:11]: Notice that and in the related msq and ark site prs i am removing the old import to the old file

**Review by ItsANameToo — APPROVED** [review]
